mlrun/feature_store/retrieval/job.py

```diff
@@ -126,9 +136,9 @@ def merge_handler(context, vector_uri, target, entity_rows=None,
         entity_rows = entity_rows.as_df()
 
     context.logger.info(f"starting vector merge task to {vector.uri}")
-    merger = LocalFeatureMerger(vector)
-    resp = merger.start(entity_rows, entity_timestamp_column, store_target, drop_columns, with_indexes=with_indexes, 
-                        query=query)
+    merger = mlrun.feature_store.retrieval.{{{engine}}}(vector, **{{{engine_args}}})
```

Comment:
```suggestion
    merger = mlrun.feature_store.retrieval.{{{engine}}}(vector, **engine_args)
```
And, to complete the suggestion - no need for str replacement here.

Reply:
Technically this would fail... Should be `**(engine_args or {})` :)